src/tasks/countChars.js

```diff
@@ -0,0 +1,17 @@
+const countChars = (str) => {
+  const info = {}
+
+  for (let i = 0; i < str.length; i++) {
+    const char = str[i]
+
```

Comment:
Why not shorten the function?
https://javascript.info/optional-chaining
`return !info[char] ? info[char] = 1  :  info[char]++`